packages/oauth/src/index.ts

```diff
@@ -725,6 +859,20 @@ interface OAuthV2Response extends WebAPICallResult {
   };
 }
 
+export interface OAuthV2ExchangeResponse extends WebAPICallResult {
+  app_id: string;
+  scope: string;
+  token_type: 'bot' | 'user';
+  access_token: string;
+  refresh_token: string;
+  expires_in: number;
+  bot_user_id?: string;
+  team: { id: string, name: string };
+  enterprise: { name: string, id: string } | null;
+  is_enterprise_install: boolean;
+  response_metadata: {}; // TODO
```

Comment:
I couldn't find what the actual shape of this property was, neither in the docs nor in conversations. If anyone happens to know, would like to update this.